### ISSUE TITLE
Fixing mov-cli's plugin and packaging issues. (mov-cli plugin package manager)

### ISSUE THREAD

**Issue by THEGOLDENPRO** [mov-cli/mov-cli#349]
> _This is currently a placeholder / draft for an issue I'm still in the middle of writing. They'll be a proper description here soon when I'm ready to drop my explanation..._

...

**Comment by ghost** [comment]
just keep it compatible with external package managers

**Comment by THEGOLDENPRO** [comment]
> just keep it compatible with external package managers

Yes that's my goal and so far package maintainers won't need to change anything, they can continue packaging plugins like normal. In my plan my idea is to prioritize plugins and dependencies installed via Linux package managers over anything in the "mov-cli package manager", giving full control to users on Linux.

So far the "mov-cli package manager" (don't have a name for it yet) exists more to help users on platforms like Windows and MacOS (exactly how will be explained in my explanation that will come above soon).
